rust/operator-binary/src/main.rs

```diff
@@ -49,6 +50,9 @@ mod built_info {
 
 pub const OPERATOR_NAME: &str = "hdfs.stackable.tech";
 
+// TODO (@NickLarsenNZ): Change the variable to `CONSOLE_LOG`
+pub const ENV_VAR_CONSOLE_LOG: &str = "HDFS_OPERATOR_LOG    ";
```

Comment:
note: Remove trailing whitespaces.

```suggestion
pub const ENV_VAR_CONSOLE_LOG: &str = "HDFS_OPERATOR_LOG";
```